en/docs/extend/errors/customize-authentication-error-messages.md

```diff
@@ -0,0 +1,28 @@
+# Customize Authentication Error Messages
+
+WSO2 Identity Server has standard error messages for different authentication errors that are encountered. See [Error Codes and
+Descriptions](../../../extend/error-codes-and-descriptions) or more information on the standard error codes and descriptions of
```

Comment:
Aren't we using subfolder in extend. For example, (../../../extend/user-mgmt/error-codes-and-descriptions)?